GraphicsForAxisEvents/src/main.ts

```diff
@@ -0,0 +1,109 @@
+import * as fs from "fs";
+import { CameraVapix } from "camstreamerlib/CameraVapix";
+import { CamOverlayAPI } from "camstreamerlib/CamOverlayAPI";
+
+type Camera = {
+  IP: string;
+  port: number;
+  user: string;
+  password: string;
+};
+
+type Settings = {
+  sourceCamera: Camera;
+  targetCamera: Camera;
+  modem: Camera;
+  events: {
+    eventName: string;
+    serviceID: number;
+    duration: number;
+  }[];
+};
+
+type Event = {
+  eventName: string;
+  duration: number;
+  co: CamOverlayAPI;
+};
+
+let cv: CameraVapix;
+let settings: Settings;
+
+function onEventMessage(event: Event) {
+  event.co.setEnabled(true);
+  setTimeout(() => {
+    event.co.setEnabled(false);
+  }, event.duration);
```

Comment:
kdyz je duration 0 tal se ti to hned vypne? tzn. duration musis specifikovat?

Reply:
Ne, duration je nepovinné. Pokud není zadané, tak se to nevypne nikdy.